src/components/Header.js

```diff
@@ -0,0 +1,13 @@
+function Header(){
+  return(
+    <header className="flex justify-between px-[25%]">
+      <h1 className="text-main-green">conduit</h1>
+      <nav className="text-main-grey">
+        <a href="#" className="ml-4">Home</a>
```

Comment:
The words are too light, they are hard to see. Please make them darker so there is more contrast. 
They should also be even darker when hovered over.

Reply:
Okay, I will make the text darker and also create a custom tailwindcss variable for hovered grey.